src/authorized-view.ts

```diff
@@ -253,3 +254,7 @@ export class AuthorizedView extends TabularApiSurface {
     }
   }
 }
+
+promisifyAll(AuthorizedView, {
+  exclude: ['initializeRow', 'generateProperties'],
+});
```

Comment:
Much like the `Table` and `Row` classes, `AuthorizedView` must use `promisify`. This is because the body of the methods in `AuthorizedView` don't return anything and they use a callback to send back results, but the methods for `AuthorizedView` should return promises like the methods for `Table` or `Row` do.

`initializeRow` and `generateProperties` are just helper methods and exist to eliminate duplicate code so should not be transformed to return promises.